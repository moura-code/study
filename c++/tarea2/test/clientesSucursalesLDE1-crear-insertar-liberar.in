crearClientesSucursalesLDE
liberarClientesSucursalesLDE
crearClientesSucursalesLDE
crearClientesABB
crearCliente 4 Rufino Ruiz 22
agregarAClientesABB
agregarAClientesSucursalesLDE
liberarClientesSucursalesLDE
crearClientesSucursalesLDE
crearClientesABB
crearCliente 5 Artemio Gonzalez 24
agregarAClientesABB
agregarAClientesSucursalesLDE
crearClientesABB
crearCliente 6 Esmeralda Gomez 30
agregarAClientesABB
agregarAClientesSucursalesLDE
liberarClientesSucursalesLDE
Fin
