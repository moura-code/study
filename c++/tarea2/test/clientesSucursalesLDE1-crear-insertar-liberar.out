Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> La SucursalesLDE de clientes ha sido creado de forma exitosa.
2> SucursalesLDE de ClientesABB liberada
3> La SucursalesLDE de clientes ha sido creado de forma exitosa.
4> El clientesABB ha sido creado de forma exitosa.
5> Cliente creado de forma exitosa.
6> Se ha agregado el cliente al clientesABB de forma exitosa.
7> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
8> SucursalesLDE de ClientesABB liberada
9> La SucursalesLDE de clientes ha sido creado de forma exitosa.
10> El clientesABB ha sido creado de forma exitosa.
11> Cliente creado de forma exitosa.
12> Se ha agregado el cliente al clientesABB de forma exitosa.
13> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
14> El clientesABB ha sido creado de forma exitosa.
15> Cliente creado de forma exitosa.
16> Se ha agregado el cliente al clientesABB de forma exitosa.
17> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
18> SucursalesLDE de ClientesABB liberada
19> Fin.
